ITS-UofIowa/sitenow_custom_profile
# Bug/188 wysiwyg btbutton

This PR fixes the issue with the Bootstrap buttons configuration being lost, resolves #188.

## Testing this PR

1. Update `system.make.yml` to use the media-oembed branch of the sitenow profile.
```yml
sitenow:
    type: profile
    download:
      url: git@github.com:ITS-UofIowa/sitenow_custom_profile.git
      branch: bug/188-wysiwyg-btbutton
```
2. Run `nile ldu`
3. Enable Bootstrap buttons for filtered_html profile
4. Add a button via the WYSIWYG
5. Save the entity
6. Edit the entity and check that the configuration for the button is the same.

## Before merging
> Before merging this pull request we need to merge ITS-UofIowa/sitenow_wysiwyg#27, create a release and update system.make.yml.
### Update
I've merged the PR and tagged a new release and updated system.make.yml, so this PR should be good to go.
